kasa/tests/fixtureinfo.py

```diff
@@ -17,6 +19,14 @@ class FixtureInfo(NamedTuple):
     data: dict
 
 
+class ComponentFilter(NamedTuple):
```

Comment:
I'd separate the changes in this file also into the tests cleanup PR.

Reply:
As per comment above needed to add this functionality to the component filter to test v1 vs greater than v1